doc/dev/quick_guide.rst

```diff
@@ -106,12 +106,12 @@ you might do something like this:
 
 Running a RadosGW development environment
 -----------------------------------------
-Add the ``-r`` to vstart.sh to enable the RadosGW
+Add the ``--num_rgw 1`` option to vstart.sh to enable the RadosGW.
 
 .. code::
 
 	$ cd build
-	$ ../src/vstart.sh -d -n -x -r
+	$ ../src/vstart.sh -d -n -x --num_rgw 1
```

Comment:
We should prefix with RGW=1 instead of using --num_rgw (which I suggest we remove!)